Account/FxAClient10.swift

```diff
@@ -491,14 +498,17 @@ open class FxAClient10 {
         return makeRequest(mutableURLRequest, responseHandler: FxADevice.fromJSON)
     }
 
-    open func oauthAuthorize(withSessionToken sessionToken: NSData, keyPair: RSAKeyPair, certificate: String) -> Deferred<Maybe<FxAOAuthResponse>> {
-        let audience = self.getAudience(forURL: self.oauthURL)
-
-        let assertion = JSONWebTokenUtils.createAssertionWithPrivateKeyToSign(with: keyPair.privateKey,
-                                                                              certificate: certificate,
-                                                                              audience: audience)
+    open func oauthAuthorize(withSessionToken sessionToken: NSData, scope: String) -> Deferred<Maybe<FxAOAuthResponse>> {
```

Comment:
This is just a convenience method that calls `RSAKeyPair.generate()` for you instead of having to pass it in.